components/cliniko/actions/get-patient/get-patient.js

```diff
@@ -0,0 +1,21 @@
+const cliniko = require("../../cliniko.app.js");
+
+module.exports = {
+  name: "Get Patient",
+  key: "cliniko-get-patient",
+  description: "Get the details of a patient by `patientId`.",
+  version: "0.0.6",
```

Comment:
We need to make this more clear in the guidelines and simpler as you develop, but when you commit these files to the GitHub repo, you'll want to set the version to `0.0.1`.